app/services/doc_auth/acuant/issuer_types.rb

```diff
@@ -0,0 +1,30 @@
+module DocAuth
+  module Acuant
+    module IssuerTypes
+      IssuerType = Struct.new(:code, :name)
+
+      UNKNOWN = IssuerType.new(0, 'Unknown').freeze
```

Comment:
Just curious: What does freezing each value here give us, when we freeze the whole array below?

Reply:
@night-jellyfish this is basically an exact copy of result_codes

Reply:
@night-jellyfish , also .freeze is not recursive, there is no deep_freeze in stock Ruby.

Reply:
Thank you for explaining!